tutorials/step_by_step/simple_2d_game.rst

```diff
@@ -55,9 +55,10 @@ is defined, then input methods that trigger it are added.
 Open the project properties dialog again, but this time move to the
 "Input Map" tab.
 
+.. role:: python(code)
```

Comment:
Oops, sorry about that.  I was testing different roles, but then realized the default one was better.  But I forgot to remove that line before committing.